docs/description.md

```diff
@@ -17,7 +17,8 @@ with
 
 In SAGA-WaTEM, an adjusted formula is used to compensate for:
 
-- Very high RUSLE-values: values above 150 are set equal to 150.
+- Very high RUSLE-values: values above 150 are set equal to 150, this value 
+  can be adapted in the code.
```

Comment:
still not sure if 'adjusted formula' is the right phrasing here. It feels more like an extra option for setting a limit to the erosion value after the calculation (this is also a post-processing step in the tool, if I remember correctly). Same for S-Value, the basic formula is used but later divided by the correction factor, if this is requested (maybe this should be made optional as well, if selected default sets to 1.4 otherwise factor 1 is used)

Reply:
ok, named this to parameters